src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,19 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (comments.length !== 0
+  ? (
+    <div className="CommentList">
+      {comments.map(comment => (
+        <CommentInfo comment={comment} key={comment.id} />
+      ))}
+    </div>
+  )
+  : (
+    <>
+      <hr />
+
+      <b data-cy="NoCommentsMessage">No comments yet</b>
```

Comment:
Using 'b' tag is not recommended as it doesn't provide any semantic meaning. Consider using 'strong', if you need to give importance to the text and for SEO reasons.